packages/shared/src/graphql/posts.spec.ts

```diff
@@ -67,11 +68,10 @@ it('should return latest changelog post', async () => {
   let queryCalled = false;
   mockGraphQL<MockFeedData>({
     request: {
-      query: SOURCE_FEED_QUERY,
+      query: LATEST_CHANGELOG_POST_QUERY,
       variables: {
         source: 'daily_updates',
         first: 1,
-        loggedIn: false,
         ranking: RankingAlgorithm.Time,
```

Comment:
Think we can simply hard set these for the changelog query so nobody will change it down the line.

Reply:
Sure, thought about it, thanks for confirming!